Create blog post on OpenTelemetry at Kubecon NA (#1814)
open-telemetry/opentelemetry.io: content/en/blog/2022/kubecon-na.md

## Other Co-located Events

[OTel Unplugged](https://www.eventbrite.com/e/otel-unplugged-kubeconcloudnativecon-detroit-2022-tickets-427595037267)
is a one-day OpenTelemetry unconference event held at Colony Club Detroit on
Tuesday, Oct 25, 2022. Join in to listen, learn about the latest features and
get involved in the project. Network with OpenTelemetry maintainers, governance
committee and technical committee members. Check out event details
[here](https://www.eventbrite.com/e/otel-unplugged-kubeconcloudnativecon-detroit-2022-tickets-427595037267).

[Open Observability Day](https://events.linuxfoundation.org/open-observability-day-north-america/)
has several OpenTelemetry sessions including panel discussions, workshops,
individual talks and lightning talks. This event will be on Tuesday Oct 25, 2022
and is co-located with Kubecon NA. The schedule of talks can be found here.

[eBPF Day](https://events.linuxfoundation.org/cloud-native-ebpf-day-north-america/)
also has some OpenTelemetry talks focused on profiling and tracing using eBPF
events. This event will be held on Tuesday, Oct 25 2022 and is co-located with

> Kubecon NA. Check out the event schedule
> [here](https://events.linuxfoundation.org/cloud-native-ebpf-day-north-america/program/schedule/).

Reviewer: ```suggestion
Kubecon NA. For details, check out the
[schedule](https://events.linuxfoundation.org/cloud-native-ebpf-day-north-america/program/schedule/).
```

The **OpenTelemetry project meeting** is to meet and network with the

> OpenTelemetry maintainers and core contributors. This meeting will be held 3-5
> pm ET on Tuesday Oct 25, 2022 in Room 336 at the

Reviewer: ```suggestion
OpenTelemetry maintainers and core contributors. This meeting will be held 3-5
PM ET on Tuesday Oct 25, 2022 in Room 336 at the
```

[conference venue](https://events.linuxfoundation.org/kubecon-cloudnativecon-north-america/venue-travel/),
Huntington Place.

## Come Visit the OpenTelemetry Project Booth

Drop by and say Hi! at the OpenTelemetry project booth in the Kubecon NA Project
Pavilion. If you’re lucky, you may even pick up some OpenTelemetry swag!

The pavilion hours are:

> Wednesday, October 26: 10:30 - 8:00 ET Thursday, October 27: 10:30 - 5:30 ET
> Friday, October 27: 10:30 - 4:00 ET

Reviewer: ```suggestion
- Wednesday, October 26, 10:30 - 20:00 ET
- Thursday, October 27, 10:30 - 17:30 ET
- Friday, October 27, 10:30 - 16:00 ET
```

We’re super excited to learn from all of you about your experience with
OpenTelemetry, how you are using OpenTelemetry, features you may need and issues
you may be running into when using OpenTelemetry.

Come join us to listen, learn and get involved in OpenTelemetry.

> See you in Detroit in three weeks!

Reviewer: ```suggestion
See you in Detroit!
```
